activities/standard-maintainance/index.md

```diff
@@ -0,0 +1,13 @@
+---
+type: index
+---
+
+# Maintainance of the Standard for Public Code
```

Comment:
The correct spelling is 'maintenance'. This needs to be fixed in the title, the filename, and anywhere it's linked. (Sorry for not spotting this much, much sooner!)

Reply:
Easy fix :) Done